app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -0,0 +1,54 @@
+module ServiceOffering
+  class AddToPortfolioItem
+    def initialize(params)
+      @req_params = params
```

Comment:
To clarify - `/req_params/params/` `:attr_reader :params`

Reply:
Sure - didn't know if we wanted to make it available after instantiating the object, but reader makes sense since they can see the params passed, but can't change them.